api/controllers/shabads.js

```diff
@@ -24,6 +25,7 @@ const allColumns = `v.ID, v.Gurmukhi, v.GurmukhiUni, v.Translations, v.PageNo AS
 const allColumnsWhere = 'AND s.ShabadID < 5000000';
 
 const error = (err, res) => {
+  console.error(err);
   res.status(400).json({ error: true, data: err });
```

Comment:
I purposely avoided expanding the scope of this PR. This will be done in a future PR for #96 and #108. I have laid the groundwork for it here because I needed it for this PR.